text/0060-tasks-api.md

```diff
@@ -150,6 +152,14 @@ List of global tasks by `type`:
 | canceledTasks | The number of tasks successfully canceled. If the task fails, `0` is displayed. `null` when the task status is enqueud or processing. |
 | originalFilters | The extracted URL query parameters used in the originating task cancelation request. |
```

Comment:
I can't edit this part...

Reply:
https://github.com/meilisearch/specifications/pull/195 It's located in the task cancelation PR